README.md

```diff
@@ -3,227 +3,156 @@
-  methods instead of view templates which is the default.
-
-### @RequestMapping
-
-This annotation defines the base URL for the API. If our base URL is
-`http://www.example.org` and we use `/api` as the value for `@RequestMapping`,
-it would create the base API as `http://www.example.org/api`.
-
-### MemberService Injection
+Now let's run this application and see what happens! Remember, to run the
+application, navigate back to the default class. In this case, the default class
+ is `SpringWebDemoApplication`. Click the green play button next to the `main`
+method.
 
-The `MemberService` object is injected automatically by Spring into the
-controller so we can use it in the class.
+The console should pop up and show something that looks like this:
 
-Now that we have talked about the class level annotations and the dependency, we
-will look at the individual details and the annotations used.
+![spring-application-started](https://curriculum-content.s3.amazonaws.com/spring-mod-1/controller/spring-boot-application-console.png)
 
-## Running the Application
+Notice that, unlike the last time we ran our default class, it does not just start
+and then immediately finish executing. It's still running!
 
-We are using Maven as the build tool and can run the following command to start
-our application:
+Open a browser and type in "http://localhost:8080/" into the URL. This is where
+our Spring application should be running. "localhost" is the standard hostname
+that a machine gives itself and "8080" is the default port that Tomcat started on.
+We can see the default port actually in the console log when we start up our
+Spring Boot application. When the browser loads the web page, we should see this:
 
-```
-./mvnw spring-boot:run
-```
+![welcome-index](https://curriculum-content.s3.amazonaws.com/spring-mod-1/controller/welcome-to-spring-boot.PNG)
 
-We can also use the globally installed `mvn` tool to run the application:
+It's the same message we return in the `index()` method within the
+`LunchController`! So what exactly happened?
 
-```
-mvn spring-boot:run
-```
+Let's look at the annotations we used!
 
-Both of these commands will start the web application on the localhost at port
-`8080`. We can append the API endpoints defined in our controller to the default
-`http://localhost:8080/` address to get the desired response from the
-application. For example, we can make a `GET` request to
-`http://localhost:8080/api/members` to retrieve a list of all the members in the
-database.
+### @RestController
 
-## Post Method
+The `@RestController` combines the following annotations:
 
-```java
-@PostMapping("/members")
-public Member createMember(@RequestBody Member member) {
-    return memberService.createMember(member);
-}
-```
+- `@Controller`: Indicates to Spring that the class represents a controller.
+- `@ResponseBody`: Configures Spring to return a JSON response from the controller
+  methods instead of view templates, which is the default.
```

Comment:
Should the students already know what a 'view template' is?

Reply:
Removed the "view template" reference and explained how it is bound to the web's response body.